operator/roles/csi-scale/templates/csi-plugin-provisioner.yaml.j2

```diff
@@ -46,6 +46,8 @@ spec:
           image: '{{ provisioner }}'
           args:
             - "--csi-address=$(ADDRESS)"
+            - "--timeout=2m"
```

Comment:
It might be useful to add a comment pointing to how and why 2m was picked.

I think this would be a good link to add in the comment:
https://github.com/kubernetes-csi/external-provisioner/tree/v1.5.0#csi-error-and-timeout-handling